tests/sentry/api/endpoints/test_project_filter_details.py

```diff
@@ -23,6 +24,50 @@ def test_put(self):
 
         assert project.get_option("filters:browser-extensions") == "1"
 
+    def test_put_health_check_filter_business(self):
+        """
+        Tests that it accepts to set the health-check filter on plans with that
+        allow it ( business plan)
```

Comment:
General nit: These tests mention business plans, but AFAICT they test whether behavior changes when the feature flag is toggled. So if we later decide to enable the feature flag based on different criteria than plans (dogfooding, EA, GA), the test case is still valid, but it won't make sense to mention plans anymore.

```suggestion
        Tests that it accepts to set the health-check filter when the feature flag is enabled
```